navigator_engine/common/graph_loader.py

```diff
@@ -91,85 +91,89 @@ def import_data(sheet_name, graphs):
-                action = model.Action(
-                    title=graph_data.at[idx, DATA_COLUMNS['ACTION']],
-                    html=action_html,
-                    skippable=skippable,
-                    complete=False
-                )
-                model.db.session.add(action)
-                model.db.session.commit()
-
-                # Parse action's resource urls and add them to database
-                resources = _parse_resources(graph_data.at[idx, DATA_COLUMNS['ACTION_RESOURCES']])
-                for resource_row in resources:
-                    resource = model.Resource(
-                        title=resource_row['title'],
-                        url=resource_row['url'],
-                        action=action
+                    action_html = _markdown_to_html(graph_data.at[idx, DATA_COLUMNS['ACTION_CONTENT']])
+                    action = model.Action(
+                        title=graph_data.at[idx, DATA_COLUMNS['ACTION']],
+                        html=action_html,
+                        skippable=skippable,
+                        complete=False
                     )
-                    model.db.session.add(resource)
+                    model.db.session.add(action)
                     model.db.session.commit()
 
-                # Add node to action
-                node_action = model.Node(
-                    ref=_get_ref(idx, 'action'),
-                    action_id=action.id
-                )
-                model.db.session.add(node_action)
-                model.db.session.commit()
+                    # Parse action's resource urls and add them to database
+                    resources = _parse_resources(graph_data.at[idx, DATA_COLUMNS['ACTION_RESOURCES']])
+                    for resource_row in resources:
+                        resource = model.Resource(
+                            title=resource_row['title'],
+                            url=resource_row['url'],
+                            action=action
+                        )
+                        model.db.session.add(resource)
+                        model.db.session.commit()
+
+                    # Add node to action
+                    node_action = model.Node(
+                        ref=_get_ref(idx, 'action'),
+                        action_id=action.id
+                    )
+                    model.db.session.add(node_action)
+                    model.db.session.commit()
 
-                graph_data.at[idx, 'DbNodeActionId'] = node_action.id
-                model.db.session.add(node_action)
-                model.db.session.commit()
+                    graph_data.at[idx, 'DbNodeActionId'] = node_action.id
+                    model.db.session.add(node_action)
+                    model.db.session.commit()
+        except Exception as e:
+            logger.error(f"Error reading row: {idx}")
+            raise e

```

Comment:
Python exception handling documentation suggests using a bare raise statement: https://docs.python.org/3/tutorial/errors.html?highlight=raise#raising-exceptions

Does this do the same thing with e.g. preserved stack trace?